completion-bundle

```diff
@@ -64,7 +66,18 @@ __bundle() {
         "" | help)
             options="help install update package exec config
                      check show outdated console open lock viz init gem
-                     platform clean doctor"
+                     platform clean doctor binstubs"
+            ;;
+        binstubs)
+            case $prev in
+            --path)
+                _filedir -d
+                return;;
+            --shebang)
+                options="ruby";;
```

Comment:
I'm 50/50 on whether completing the default value for SHEBANG makes any sense. On one hand, it's instructive. But OTOH, if you intend to use the "ruby" shebang, then there's no reason to pass the option+arg in the first place.

So maybe I should just remove these two lines? Either way, I suppose we'd want the behavior the match with `install --shebang` completion.